src/NearestNeighbors.jl

```diff
@@ -1,11 +1,12 @@
-__precompile__()
+# __precompile__()
```

Comment:
Yeah, it is just commented out because it is a WIP so I dont have to wait for recompilation :)